### ISSUE TITLE
Make Misago to look like old ubuntu.lt

### ISSUE THREAD

**Issue by sirex** [python-dirbtuves/ubuntu.lt#5]
Update Misago styling to make it look like old ubuntu.lt.

**Comment by sirex** [comment]
@narunask I think this is low priority at the moment?

**Comment by narun4sk** [comment]
I agree about the priority. I totally disagree about pulling it back to the old looks...
Yes, some styling should be preserved, but I vote for the more minimalistic styling, something similar to what misago has, but adopted to our needs where possible.

**Comment by sirex** [comment]
I also agree with you, I like how Misago looks it is far better than current ubuntu.lt/forum design. But small elements to resemble ubuntu.lt look could be added for sure.

So lets keep this user story in ice box for now.

**Comment by psukys** [comment]
What about using the [Ubuntu web style guidelines](http://design.ubuntu.com/web-style-guide), as this ubuntu.lt is a lithuanian ubuntu community?

**Comment by narun4sk** [comment]
Good find. I think we should definitely stick at least to some of these guidelines.
